src/lib/services/types/wasm/contract.ts

```diff
@@ -191,17 +193,22 @@ export const zMigrationHistoriesResponseItemLcd = z
     operation: z.nativeEnum(RemarkOperation),
```

Comment:
let's create `zRemarkOperation` in `lib/types/contract.ts`, then use it in `zContractHistoryRemark` and here